### PR TITLE
Add Arm64 encoding for IF_SVE_BL_1A

### PR DESCRIPTION
**Pull request by a74nh**
These instructions use a pattern to specify which elements to count from a predicate. This is encoded as 5 bits in the instruction. I've added a new entry in `instrdesc` union (to prevent it increasing the size).

Imm encoding is a 4bit number representing 1 to 16, so 1 needs subtracting when encoding.

New clr test output (identical to capstone):
```
cntb x0, pow2
cntd lr, vl1, mul #16
cnth x12, vl7, mul #5
cntw x23, vl256, mul #7
cntb x21, mul4, mul #8
cntd x15, mul3, mul #10
cnth x5, all, mul #13
```

### CODE REVIEW

**Comment by a74nh** [comment]
@kunalspathak @dotnet/arm64-contrib

**Review by TIHan — APPROVED** [review]
This looks good imo. I see why you have to introduce the new entry because you have to flow that info to the part of the actual encoding.

I'd wait for Kunal's review too before merging.

File: src/coreclr/jit/emit.h
Comment (kunalspathak) [inline src/coreclr/jit/emit.h:991]: this looks ok to me, but wondering why not just have something like this after the end of `struct` definition.

```c++
#ifdef TARGET_ARM64
  insSvePattern _idSvePattern;
#endif
```

**Review by kunalspathak — COMMENTED** [review]
A minor comment. Otherwise, LGTM

File: src/coreclr/jit/emit.h
Comment (a74nh) [reply src/coreclr/jit/emit.h:991]: Happy to switch to that.

My reasoning was when you look at the whole file we have an `#ifdef` for each target. Many of them just have the same thing (`_idReg3` and `_idReg4`). The nesting of the Arm32/Arm64 is a little messy to read and doesn't fit the rest of the file style. Splitting it made it simpler.

**Review by kunalspathak — APPROVED** [review]
LGTM

**Pull request merged**
